lighthouse-core/config/experimental-config.js

```diff
@@ -14,14 +14,28 @@
 const config = {
   extends: 'lighthouse:default',
   audits: [
-    'full-page-screenshot',
+    'byte-efficiency/duplicated-javascript',
```

Comment:
looks like merge messed up–these audits are no longer in experimental.

delete the latter `audits` and put `valid-source-maps` here

Reply:
yeah it got messed up during rebase

I reset it to the what the experimental-config looks like in `master` if that's okay